docs/tsg_install.md

```diff
@@ -60,6 +60,27 @@ iscsiadm version 2.0-874
 
 To resolve this issue, do not install `open-iscsi / iscsi-initiator-utils` on the host nodes when using the Rancher Container Engine (RKE).
 
+## How can I do selective in disks for creating storage pool using cStor?
+
+With latest 0.7.0 OpenEBS release, following type of disk types/paths are excluded by NDM to identify the disks to create cStor Pool on nodes. 
```

Comment:
With the latest OpenEBS 7.0 release, the following disk types/paths are excluded by NDM which identifies the disks to create cStor pools on nodes.